torch/nn/_functions/bilinear.py

```diff
@@ -0,0 +1,66 @@
+import torch
+from torch.autograd import Function
+
+
+class Bilinear(Function):
+
+    def forward(self, input1, input2, weight, bias=None):
+        self.save_for_backward(input1, input2, weight, bias)
+
+        output = input1.new()
+
+        self.buff2 = input1.new()
```

Comment:
If we don't have a problem with redefining it in backward then I guess I can do it.